soccer/gameplay/plays/Clear/adaptive_clear.py

```diff
@@ -0,0 +1,280 @@
+import main
+import robocup
+import behavior
+import constants
+import enum
+
+import standard_play
+import evaluation.ball
+import evaluation.passing_positioning
+import tactics.coordinated_pass
+import tactics.defensive_forward
+import tactics.simple_zone_midfielder
+import tactics.advance_zone_midfielder
+import skills.move
+import skills.capture
+import situational_play_selection
+
+
+
+#This is essentially a copy of adaptive formation, found in the legacy folder,
+#With the shooting option removed. It is intended to be used as a clearing play
+class AdaptiveClear(standard_play.StandardPlay):
```

Comment:
FWIW, this has both passing and "clearing" from adaptive. It doesn't make much of a difference, but yeah.